### PR TITLE
chore: upgraded versions.tf to include minor bumps from tpg v5

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

**Comment by imrannayer** [comment]
@g-awmalik there seems to be something wrong with API. Cross project load balancer feature was working fine back in August when I released 9.2. Check PR #348.

File: examples/cross-project-mig-backend/main.tf [hunk 1]
```diff
@@ -27,7 +27,7 @@ module "gce-lb-http" {
 
   backends = {
     default = {
-      project     = var.service_project
+      project     = var.host_project
```
Comment (imrannayer) [inline examples/cross-project-mig-backend/main.tf:30]: @g-awmalik this change is throwing error. Can you plz revert this change.

**Comment by imrannayer** [comment]
@g-awmalik I tested it using CLI and received same error.

**Comment by imrannayer** [comment]
Already fixed in #374

**Comment by joemiller** [comment]
Yes! Will a new version of the module be cut soon? tnx
